http/server.lua

```diff
@@ -1296,6 +1296,12 @@ local function url_for_httpd(httpd, name, args, query)
     end
 end
 
+local VERIFY_CLIENT_OPTS = {
+    off = sslsocket.SET_VERIFY_FLAGS.SSL_VERIFY_NONE,
+    optional = sslsocket.SET_VERIFY_FLAGS.SSL_VERIFY_PEER,
+    on = sslsocket.SET_VERIFY_FLAGS.SSL_VERIFY_PEER + sslsocket.SET_VERIFY_FLAGS.SSL_VERIFY_FAIL_IF_NO_PEER,
```

Comment:
Does some bit operation is actually expected to be here? There is a proper `bit` library for it.